docs/tutorial/installation.md

```diff
@@ -36,6 +36,17 @@ In addition to changing the architecture, you can also specify the platform
 npm install --platform=win32 electron
 ```
 
+## Skip binary download
+When installing electron for the first time using npm, it automatically downloads the electron binary.
+
+This can sometimes be unnecessary, e.g. in a CI environment, when testing another component.
+
+To prevent the binary from being downloaded when you install all npm dependencies just set the environment variable `ELECTRON_BINARY_SKIP_DOWNLOAD`.
```

Comment:
```suggestion
To prevent the binary from being downloaded when you install all npm dependencies you can set the environment variable `ELECTRON_BINARY_SKIP_DOWNLOAD`.
```